### ISSUE TITLE
Fix batch normalization with momentum

### ISSUE THREAD

**Issue by asuhan** [pytorch/xla#220]
Right now we're not using the momentum in the XLA lowering.
